muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/evaluation/body/application/evaluation/evaluation-journal-event.tsx

```diff
@@ -291,6 +301,27 @@ const EvaluationJournalEvent: React.FC<EvaluationDiaryEventProps> = (props) => {
       </div>
 
       <AnimateHeight duration={400} height={showContent ? "auto" : 0}>
+        {material && (
+          <>
+            <div className="evaluation-modal__item-body-journal-material-toggle">
```

Comment:
Onko tosiaan pakollista tehdä uusi className evaluation-modal:in alle? Uskoisin että siellä olisi jo olemassa jokin geneerinen item className jota voisi käyttää tässäkin tapauksessa koska `evaluation-modal__item-body-journal-material-toggle` css luokassa ei ole kummia tyyliasetuksia eikä sitä edes välttämättä tarvitsisi. Tätä elementtiä käytetään kuitenkin vain button elementin wrapperinä joka ottaa oman rivin käyttöönsä. Perus div ilman classNamea tekee sen saman.